index.html

```diff
@@ -11,54 +11,55 @@
                 </form>
             </div>
             <!--filters-->
-            <div id="filter">
-                <h2>Filtres</h2>
-                <div>
-                    <button class="btn" onclick="filterSelection('Economique')"> <i class="fa-solid fa-money-bill-wave icon"></i> Economique</button>
-                    <button class="btn" onclick="filterSelection('Familial')"> <i class="fa-sharp fa-solid fa-child-reaching icon"></i> Familial</button>        
-                    <button class="btn" onclick="filterSelection('Romantique')"> <i class="fa-solid fa-heart icon"></i> Romantique</button>
-                    <button class="btn" onclick="filterSelection('Animaux autorisés')"> <i class="fa-solid fa-dog icon"></i> Animaux autorisés</button>
+            <div class="filter">
+                <h2 class="filter__title">Filtres</h2>
+                <div class="filter__btn">
+                    <button class="filter__btn--style" onclick="filterSelection('Economique')"> <i class="fa-solid fa-money-bill-wave icon"></i> Economique</button>
+                    <button class="filter__btn--style" onclick="filterSelection('Familial')"> <i class="fa-sharp fa-solid fa-child-reaching icon"></i> Familial</button>        
+                    <button class="filter__btn--style" onclick="filterSelection('Romantique')"> <i class="fa-solid fa-heart icon"></i> Romantique</button>
+                    <button class="filter__btn--style" onclick="filterSelection('Animaux autorisés')"> <i class="fa-solid fa-dog icon"></i> Animaux autorisés</button>
                 </div>
             </div>
-            <!--infos-->
-            <div id=info>
-                <i class="fa-solid fa-info icon" id="circle-info"></i>
-                <p>Plus de 500 logements sont disponibles dans cette ville</p>
+            <!--info-->
+            <div class=info>
+                <i class="info__item fa-solid fa-info icon"></i>
+                <p class="info__text">Plus de 500 logements sont disponibles dans cette ville</p>
             </div>
             <!--cards-->
-            <div id="staysandpopular">
+            <div class="staysandpopular">
                 <!--stays in marseille-->
-                <section id="stays">
+                <section class="stays">
                     <div class="head">
                         <h2>Hébergements à Marseille</h2>
                    </div>
                     <div class="choices">
                         <a href="#">
                             <article class="elements-card">                    
                                 <div class="card-image">
-                                    <img src="images/hebergements/canebiere.jpg" alt="Auberge de la Canebière">
+                                    <img src="images/hebergements/canebiere.webp" alt="Auberge de la Canebière" loading="lazy">
                                 </div>
```

Comment:
Pour les image ajouté le width and height pour evité le drop d'image quand elle charge

Reply:
width et height ajoutées